docs/openapi/components/schemas/credentials/VerifiableBusinessCard.yml

```
@@ -229,7 +229,31 @@ properties:
       - worksFor
       - jobTitle
   proof:
-    $ref: ../snippets/proof.yml
+    type: object
+    properties:
+      type:
+        type: string
+        description: Signature suite for the proof
+        enum:
+          - Ed25519Signature2018
+      created:
+        description: Creation timestamp for the proof in the form of an XML datestring
+        type: string
+      verificationMethod:
+        description: The fragment from which the public key can be de-referenced, in the form of a URI
+        type: string
+      proofPurpose:
+        description: In the case of credentials, the proof should be the constant, 'assertionMethod'
```

```suggestion
        description: For credentials, the proof should be the constant, 'assertionMethod'
```
What about otherwise?

---

VPs / "Object Capabilities" use different proof types.

---

Perhaps someone could draft another sentence or two for the description, which go/es into the non-credential case(s).

---

Perhaps we just link to the spec: https://w3c.github.io/vc-data-integrity/#proof-purposes

---

```suggestion
        description: For credentials, the `proofPurpose` should be the constant, 
        `'assertionMethod'`. For other uses, see [[[[whatever the citation format 
        is... "the Proof Purposes section of the Data Integrity spec"]]]]
```